include/utils.h

```diff
@@ -18,8 +18,12 @@ limitations under the License.
 #pragma once
 
 #include <Python.h>
+
+#if PY_VERSION_HEX < 0x30C00F0  // Python 3.12.0
+#include <structmember.h>       // PyMemberDef
+#endif
```

Comment:
We only need to include `Python.h` since 3.12.0a3. References:

- python/cpython#47146
- python/cpython#99014